src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,14 @@
+package duke;
+
+/**
+ * Class that handles Duke Exceptions
+ */
+public class DukeException extends Exception {
+    /**
+     * Constructor for Duke Exception
+     * @param s custom error message when a Duke Exception is thrown
```

Comment:
Do:
1. add a line between the description and the @param
2. add punctuation
```
/**
 * Constructs a Duke Exception.
 *
 * @param s Custom error message when a Duke Exception is thrown.
 */
```

https://se-education.org/guides/conventions/java/index.html#comments
"Empty line between description and parameter section"
"Punctuation behind each parameter description"

Reply:
ok thanks!